queries.sql

```diff
@@ -23,3 +23,71 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT name, species FROM animals;
+
+COMMIT;
+
```

Comment:
Great job @jkanyi-web 🤝

- [ ]  As stated in the project requirements, kindly `verify that change persists after commit`.